cachito/web/purl.py

```diff
@@ -101,6 +100,33 @@ def _to_purl_pip(package):
         return f"pkg:generic/{name}?download_url={quoted_url}&checksum={checksum}"
 
 
+def _to_purl_rubygems(package):
+    if package.version and package.version.startswith("./"):
+        # Package is relative to the parent module
+        normpath = os.path.normpath(package.version.removeprefix("./"))
```

Comment:
AFAIK `normpath` removes the `./` prefix already

Reply:
you're right